docs/guide/advocacy.md

```diff
@@ -0,0 +1,83 @@
+# Advocacy
+
+Much of e18e's work involves making changes in existing open source repositories.
+These repositories generally have their own workflows that their maintainers expect to adhere to.
+It's important when suggesting changes to repositories to respect those workflows and actively work with those maintainers.
+
+## Coordinating Issues
+
+Many repositories prefer discussing potential changes in an issue before they can fully review a pull request.
+Unless a repository maintainer has explicitly indicated a willingness to bypass issue practices, stick with their normal flow of filing an issue and fully filling out the most applicable issue template.
+[Explaining Changes](#explaining-changes) covers the common information you'll want to include.
+
+### Umbrella e18e Tracking
+
+When many changes are possible for a single repository, please search for an "umbrella" issue in [e18e/ecosystem-issues](https://github.com/e18e/ecosystem-issues), and open one if it doesn't already exist.
+That will help with tracking progress and raising visibility for other e18e participants.
+
+We also use these issues to keep track of maintainer preferences around the initiative.
+If a maintainer requests a preference, especially one to not be sent e18e pull requests, please note it there.
+
+## Explaining Changes
+
+Most maintainers are not active participants in e18e.
+Many will have not heard of the initiative in the first place.
```

Comment:
this feels like an odd bit of wording to include as it depends on the circle and will change over time

e.g. much of the vue/vite community is aware of e18e, including maintainers. similarly, many of the svelte community's maintainers are too.

it'd probably be better to word this as "Some maintainers are not active participants in e18e or have not heard of it"